src/Sqs/Input/CreateQueueRequest.php

```diff
@@ -69,15 +69,24 @@ public function gettags(): array
     public function requestBody(): array
     {
         $payload = ['Action' => 'CreateQueue', 'Version' => '2012-11-05'];
-        if (null !== $this->QueueName) {
-            $payload['QueueName'] = $this->QueueName;
-        }
-        if (null !== $this->Attributes) {
-            $payload['Attribute'] = $this->Attributes;
-        }
-        if (null !== $this->tags) {
-            $payload['Tag'] = $this->tags;
-        }
+        $indices = (object) [];
```

Comment:
What about? 
`$indices = new \stdObject()`

Isnt this the same?

Reply:
fixed